ui/src/app/core/notify-loading.service.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  Component,
```

Comment:
It looks like several of these imports are not being used and can be removed including `Component`, `ElementRef`, `OnInit`, `ViewChild` and `{Observable}`

Reply:
Done.